<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationDialog</class>
 <widget class="QDialog" name="OperationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="dictButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Słowniczek</string>
   </property>
  </widget>
  <widget class="QTableView" name="operationsTableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>971</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label1">
      <property name="text">
       <string>Nazwa kontahenta</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="invoiceLabel">
      <property name="text">
       <string>Dowód</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="invoiceEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="dateEdit1">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="dateEdit2">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="dateEditLabel1">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="dateEditLabel2">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label2">
      <property name="text">
       <string>Identyfikator podatkowy</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="idLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="contractorLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>651</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="kGridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addInvoiceButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj fakture</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
